wgpu-core/src/command/compute.rs

```diff
@@ -230,7 +235,7 @@ impl<G: GlobalIdentityHandlerFactory> Global<G> {
                         state
                             .binder
                             .reset_expectations(pipeline_layout.bind_group_layout_ids.len());
-                        let mut is_compatible = true;
+                        let mut is_compatible = true; // TODO: How do push constants influence this value
```

Comment:
I think we can remove the TODO. The idea is that we stop trying to rebind descriptors after the first bind group layout mismatch (i.e. ID doesn't match the expected layout). But for push constants, it's never a case where the data is not compatible with the layout, so they don't affect this.

Reply:
ping

Reply:
still there!

Reply:
Haven't pushed yet :wink: